tests/test_client_session_schema.py

```
@@ -0,0 +1,498 @@
+    wrapper.client = client
+    wrapper._session_path = MagicMock(return_value="synthetic/session/user_session")
+    wrapper._init_client = MagicMock()
+
+    with patch.object(
+        client_module.TelegramClientWrapper,
+        "disconnect_and_clear_cache",
+        new=AsyncMock(),
+    ) as disconnect_and_clear_cache:
+        try:
+            asyncio.run(wrapper.ensure_connected())
+            assert False, "expected wrong session error to be re-raised"
```

Ruff (0.15.12)

[warning] Do not `assert False` (`python -O` removes these calls), raise `AssertionError()`

Replace `assert False`

(B011)